ssl/quic/quic_channel_local.h

```diff
@@ -131,15 +141,29 @@ struct quic_channel_st {
      */
     QUIC_CONN_ID                    retry_scid;
 
-    /* The DCID we currently use to talk to the peer and its sequence num. */
+    /*
+     * The DCID we currently use to talk to the peer and its sequence num.
+     *
+     * TODO(QUIC FUTURE) remove the second two, both are contained in
+     * srt_list_seq (defined below).
```

Comment:
I am not sure we would ever want to remove these members.

Reply:
When we starting using more than one CID for transmission, `cur_remote_seq_num` will be meaningless.  So I can see a definite case for it.  The case for `cur_retire_prior_to` is weaker.

As is stands we're duplicating data and we will need to maintain consistency.  The old adage applies:

> There are 2 hard problems in computer science: cache invalidation, naming things, and off-by-1 errors.

:stuck_out_tongue_winking_eye: 

I'll update the wording to _consider removing_.